modules/db_news.py

```diff
@@ -104,7 +108,8 @@ def draw_news_mod(news_s_x, news_s_y, the_news, color, draw):
 def run_news_mod(NEWS_URL, NEWS_API, NEWS_SOURCES,
                  news_country,  mod_t_s_x, mod_t_s_y, draw, color):
     """Call functions to get and display news."""
-    news_array = get_news(NEWS_URL, NEWS_API, NEWS_SOURCES, news_country, color)
+    news_num = 0		#FIXME - this values was missing in the get_news call below.  Not sure where to get it.  dashboard.py appears to call this twice, once set to 0, other set to 1.

```

Comment:
yeah, it calls once per the sources selected and another time for your country, hardcoding it here would always bring country news based on geolocation

Reply:
I see the problem, so, since I wanted to contain the steps in modules, 
1-build the template
2-get the info 
3-paint the info

it wasnt working well when pulling 2 sets of info between modules, or at least I did had the knowledge
I ended up doing that directly in the dashboard.py

the function run_news_mod ended not being used anymore but stayed there for future comeback